pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -132,3 +148,11 @@ def makevector(*x):
         return torch.tensor(x, dtype=torch_dtype)
 
     return makevector
+
+
+@pytorch_funcify.register(TensorFromScalar)
+def pytorch_funcify_TensorFromScalar(op, **kwargs):
+    def tensorfromscalar(x):
+        return x
```

Comment:
The Op should create a tensor
```suggestion
        return torch.as_tensor(x)
```